src/solver.jl

```diff
@@ -253,10 +253,12 @@ Result with 2 solutions
 
     solve(F; start_system = :polyhedral, only_torus=false, options...)
 
-Solve the system `F` using a start system computed from the Newton Polytopes of the
-entries of `F`. The number of paths to track is equal to the mixed volume of the
+Solve the system `F` in two steps: first solve a generic system computed from
+the Newton Polytopes of the entries of `F`, then perform a coefficient
+homotopy towards `F`. The number of paths to track is equal to the mixed volume of the
```

Comment:
I found the information from #316 useful, so I added them to the docstring. This docstring is becoming quite long, it might be worthwhile to move to its own file under `docs/` instead. What do you think?

Reply:
Yeah the docstring is really already painfully long, however it is quite handy to an overview of the different arguments available view the help menu.
I think a good solution would be to have a denser docstring (while still mentioning the most important arguments) and a more extensive documentation for solve in the documentation